src/components/header/drawer/index.tsx

```diff
@@ -0,0 +1,84 @@
+import React, { FC } from 'react';
+
+import { Box, DrawerBody } from '@chakra-ui/react';
```

Comment:
Can you use our component `Box` instead importing it through '@chakra-ui/react'? Thanks